### PR TITLE
Split README into mdbook documentation

### PR DESCRIPTION
**Pull request by vadorovsky**
README was getting really long. This change moves specific information
about building and using PacketStreamer to separate mdbook subpages.

It also adds information about using PacketStreamer with Suricata.

Fixes: #4
Signed-off-by: Michal Rostecki <vadorovsky@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,278 +1,76 @@
...
-### Receiver side
+PacketStreamer **sensors** are started on the target servers. Sensors capture
+traffic, apply filters, and then stream the traffic to a central reciever.
+Traffic streams may be compressed and/or encrypted using TLS.
 
-```bash
-cd PacketStreamer
-./packetstreamer receiver --config ./contrib/config/receiver-vagrant.yaml
-```
+The PacketStreamer **reciever** accepts PacketStreamer streams from multiple
```
Comment (noboruma) [inline README.md:24]: typo: receiver ;)

File: README.md [hunk 2]
```diff
@@ -1,278 +1,76 @@
...
-# Get in touch
+## Get in touch
 
 Thank you for using PacketStreamer.
 
 * [<img src="https://img.shields.io/badge/slack-@deepfence-brightgreen.svg?logo=slack">](https://join.slack.com/t/deepfence-community/shared_invite/zt-podmzle9-5X~qYx8wMaLt9bGWwkSdgQ) Got a question, need some help?  Find the Deepfence team on Slack
-* https://github.com/deepfence/PacketStreamer/issues: Got a feature request or found a bug?  Raise an issue
-* [productsecurity at deepfence dot io](SECURITY.md): Found a security issue?  Share it in confidence
+* https://github.com/deepfence/PacketStreamer/issues: Got a feature request or found a bug? Raise an issue
+* [productsecurity at deepfence dot io](SECURITY.md): Found a security issue? Share it in confidence
```
Comment (noboruma) [inline README.md:64]: Shall we reuse the same format as on line 69?
`*at* and *dot*`

**Review by noboruma — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vadorovsky) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (vadorovsky) [reply README.md]: fixed

**Pull request merged**